src/components/NcPopover/NcPopover.vue

```diff
@@ -565,7 +565,11 @@ export default {
 			this.addEscapeStopPropagation()
 		},
 		afterHide() {
-			this.getPopoverContentElement().addEventListener('transitionend', () => {
+			/**
+			 * Component might be unmounted before 'apply-hide' is emitted.
+			 * Optional chaining is added to prevent errors in this case.
+			 */
+			this.getPopoverContentElement()?.addEventListener('transitionend', () => {
```

Comment:
If we don't expect `<NcPopover>` to trigger `after-hide` on unmounting:

```suggestion
			// On component unmounting Vue:
			// 1. Resets the template ref to null
			// 2. Triggers child's `beforeUnmount`
			//    - On this moment <Dropdown> instance still exists and emits the `afterHide` event
			// 3. Actually removes the child node
			// 4. Triggers child's `unmounted`
			// 
			// Thus when the last `apply-hide` is triggered during unmounting, the template ref is null already (even the child <Dropdown> is still mounted), and we cannot getPopoverContentElement
			// 
			// In this case we don't need to trigger `after-hide` and can ignore it
			this.getPopoverContentElement()?.addEventListener('transitionend', () => {
```

Reply:
If we do expect `<NcPopover>` to trigger `after-hide` on unmounting:

```suggestion
			// On component unmounting Vue:
			// 1. Resets the template ref to null
			// 2. Triggers child's `beforeUnmount`
			//    - On this moment <Dropdown> instance still exists and emits the `afterHide` event
			// 3. Actually removes the child node
			// 4. Triggers child's `unmounted`
			// 
			// Thus when the last `apply-hide` is triggered during unmounting, the template ref is null already (even the child <Dropdown> is still mounted), and we cannot getPopoverContentElement
			
			/* handle it some other way */
```

Reply:
Update: in `stable8` there is no `after-hide` on unmounting, because `transitionend` with the event triggering happens after the component is unmounted already, so emit does nothing.

Reply:
So let's ignore it when undefined, and if we will want the even later, let's add it in both versions.